src/legos/Button/Button.js

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+import { Button } from "grommet";
+ 
+export const StyledButton = styled(Button)`
```

Comment:
Please remove that yellow border around the button when it's clicked. 
<img src="https://user-images.githubusercontent.com/12502321/82294956-9b318480-99b7-11ea-9d19-4956f0fb19dd.png" width="400">